nb/projects/BarbourSmith/belt_sprockets/belt_sprockets.md

```diff
@@ -7,9 +7,8 @@ const additionalToothDepth = 0; //mm
 ```
 
 ```JavaScript
-const ToothSpaceingCurvefit = (teeth, b, c, d) => {
-  return ((c * Math.pow(teeth, d)) / (b + Math.pow(teeth, d))) * teeth;
-};
+const ToothSpaceingCurvefit = (teeth, b, c, d) =>
```

Comment:
This is just a style change right? No functional difference here?

Reply:
Yeah.